client/disputes/evidence/index.js

```diff
@@ -133,28 +134,53 @@ export const DisputeEvidenceForm = ( props ) => {
 			isLoading,
 			isDone,
 			error,
-			help: expandHelp( field.description ),
+			help: readOnly ? '' : expandHelp( field.description ),
```

Comment:
IMO, help tooltips should remain even if we are in read-only mode.

Reply:
Marking this as resolved, since it is along the same lines as the [conversation above](https://github.com/Automattic/woocommerce-payments/pull/4840#discussion_r990986152). Thanks for your review and guidance, Vlad!